pyobas/signatures/signature_match.py

```diff
@@ -0,0 +1,12 @@
+from pyobas.exceptions import OpenBASError
+from pyobas.signatures.types import MatchTypes
+
+
+class SignatureMatch:
+    def __init__(self, match_type: str, match_score: int):
+        if not match_score and match_type != MatchTypes.MATCH_TYPE_SIMPLE:
+            raise OpenBASError(
```

Comment:
Maybe use: if match_score is None ? what happens if match_score is 0? not 0 === not None => true, non? match_score could be 0?

Reply:
Yes good spot